primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/PrimeSelenium.java

```diff
@@ -127,6 +127,14 @@ public static void goTo(AbstractPrimePage page) {
              */
             wait(500);
         }
+        else if (isChrome()) {
+            /*
+             * Chrome (since 128) has sometimes (unpredictable) "org.openqa.selenium.NoSuchElementException: no such element: .."
+             * for the first Selenium-operation after page-load.
+             * So try to wait a bit. Or maybe find a better check to detect page-load is finished.
+             */
+            wait(200);
```

Comment:
For now it's a work-around. Maybe we find a solution for the root-cause at some point in the future.

Reply:
now we have a clean solution :-)